### PR TITLE
Replace references of openjdk-systemtest with aqa-systemtest

### PR DESCRIPTION
**Pull request by Mesbah-Alam**
Related to https://github.com/adoptium/aqa-systemtest/issues/424

Signed-off-by: Mesbah_Alam@ca.ibm.com <Mesbah_Alam@ca.ibm.com>

### CODE REVIEW

**Comment by Mesbah-Alam** [comment]
Works fine in non-parallel build. Sanity.jck green: Test_grinder/job/Grinder/14986/

**Review by karianna — APPROVED** [review]
LGTM - take out of draft?

**Comment by llxia** [comment]
>Works fine in non-parallel build

To test parallel builds, please regenerate `Grinder_testlist_*`. Or delete them if you do not need the history. Some `Grinder_testlist_* ` job is generated with `Lightweight checkout` set to true, so it uses Adopt master for pipeline changes.

**Comment by Mesbah-Alam** [comment]
Tests: 
- sanity.jck (with parallel = 6) - Grinder/15334/ - green
- extended.system (with parallel = 4) - Grinder/15328/ - green

**Comment by llxia** [comment]
Please note all 4 PRs have to merged together:
adoptium/STF#115
adoptium/aqa-systemtest#425
eclipse-openj9/openj9-systemtest#129
adoptium/aqa-tests#2542

**Review by llxia — APPROVED** [review]
LGTM

**Comment by karianna** [comment]
I think eclipse-openj9/openj9-systemtest#129 is meant to be eclipse-openj9/openj9-systemtest#132

Also, putting this into draft so it's not merged by accident until we're ready.

File: stf.build/include/top.xml [hunk 1]
```diff
@@ -322,7 +322,7 @@ limitations under the License.
 	</target>
 
 	<!--
-		Some tests may require test.debugging requires tools.jar for building (e.g. test.debugging in openjdk-systemtests).
+		Some tests may require test.debugging requires tools.jar for building (e.g. test.debugging in aqa-systemtests).
```
Comment (smlambert) [inline stf.build/include/top.xml]: predates this change, but there is no 's' on the end of aqa-systemtest

**Review by smlambert — APPROVED** [review]
aside from minor spelling in comment, LGTM

File: stf.build/include/top.xml (hunk 1 above)
Comment (Mesbah-Alam) [reply stf.build/include/top.xml]: Thanks for spotting. Fixed!

**Pull request merged**